tests/e2e/config.js

```diff
@@ -8,6 +8,7 @@ const OUTPUT_DIR = process.env.WORKING_DIRECTORY || './tests/e2e/results';
 // add your test name here …
 const TEST_NAMES = {
     AppStartTime: 'App start time',
+    OpenSearchPage: 'Open search page',
```

Comment:
```suggestion
    OpenSearchPage: 'Open search page TTI',
```

Maybe we should add TTI? Just to help with the test names as discussed?